samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -52,6 +60,8 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       adapter.onStateChange((state: CallAdapterState) => {
         const pageTitle = convertPageStateToString(state);
         document.title = `${pageTitle} - ${webAppTitle}`;
+
+        setCallId(state?.call?.id);
```

Comment:
nit: The reason to use useRef instead of setState is this will trigger a re-render (when all we want is to log something new). As this is the sample app wouldn't worry about it teenie optimization

Reply:
Alright cool can make the change.